app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
```

Comment:
format your html
it should be
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/56816494/130720234-007351f2-0320-4314-9898-b97aa14316e1.png)